src/config.py

```diff
@@ -26,7 +26,7 @@
 OUTPUT_DIR = config('OUTPUT_DIR', default=(BASE_DIR / 'output'), cast=Path)
 WRDS_USERNAME = config("WRDS_USERNAME", default="")
 
-START_DATE = config("START_DATE", default="2007-01-01")
+START_DATE = config("START_DATE", default="2022-01-01")
```

Comment:
This is only in order to have a short computing time (retrieving the date, merging...). It will be changed at the end of the project to get the full data. The tests related to load and merge might need an update at that time.